Sources/Client/Views/Play/PlayServerView.swift

```diff
@@ -23,7 +23,14 @@ struct PlayServerView: View {
   var serverDescriptor: ServerDescriptor
   var renderCoordinator: RenderCoordinatorProtocol
   
-  init(serverDescriptor: ServerDescriptor, resourcePack: ResourcePack, inputCaptureEnabled: Binding<Bool>, delegateSetter setDelegate: (InputDelegate) -> Void) {
+
+  // Link whether the cursor is captured to whether input gets sent to delegate
+  init?(serverDescriptor: ServerDescriptor, resourcePack: ResourcePack?, inputCaptureEnabled: Binding<Bool>, delegateSetter setDelegate: (InputDelegate) -> Void) {
```

Comment:
don't use optional resource pack. PlayServerView should never be created without a resource pack loaded

Reply:
it should just be impossible to get the ui into a state it shouldn't when the resource pack isn't loaded (like it was before)

Reply:
Agreed on this, imo loadingState will def need to be refactored then
Will fix 👍